networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -6,4 +6,23 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+As a cluster administrator, you can configure an additional network for a virtual routing and forwarding (VRF) domain by using the CNI VRF plugin. The virtual network created by this plugin is associated with a physical interface that you specify. 
```

Comment:
Spell out CNI on first appearance?

Reply:
```suggestion
As a cluster administrator, you can configure an additional network for a virtual routing and forwarding (VRF) domain by using the CNI VRF plugin. The virtual network that this plugin creates is associated with the physical interface that you specify. 
```

Reply:
Regarding CNI, I am tempted to leave this alone as it's part of the plugin name? Container Network Interface (CNI) VRF plugin is a bit confusing?

Reply:
oh yeah if the name is the abbreviated version that makes sense to me so the user will find it :+1: